__tests__/app.test.js

```diff
@@ -0,0 +1,46 @@
+const seed = require("../db/seeds/seed.js");
+const { categoryData, commentData, reviewData, userData } = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+const request = require("supertest");
+const app = require("../index.js");
+
+
+beforeEach(() => seed({ categoryData, commentData, reviewData, userData }));
+
+afterAll(() => db.end());
+
+describe("404 endpoint", () => {
+    test("If invalid url is sent, output 404 status code and a message", () => {
+        return request(app)
+            .get("/api/invalid_url")
+            .expect(404)
+            .then(({ body: { msg } }) => {
+                expect(msg).toBe("Invalid Path");
+            });
+    });
+});
+
+describe("GET Categories", () => {
+    const testCategory = {
+        slug: expect.any(String),
+        description: expect.any(String)
+    }
+    test("Test that /api/categories returns 200", () => {
```

Comment:
This is an unncessary test as you are testing for it in the next test also. For your endpoint tests it will be okay to combine status code tests with the other tests